### PR TITLE
doc: Fix syntax errors in example JSON

### PR DESCRIPTION
**Pull request by thomshutt**
**What does this pull request do? Explain your changes. (required)**
Adjust sample config JSON to be valid and usable

**Specific updates (required)**
* Remove trailing commas
* FPS has to be an integer
* Profile has to be a string

**How did you test each of these updates (required)**
Started local Broadcaster using this config

**Does this pull request close any open issues?**
No

**Checklist:**

- [x] Read the [contribution guide](./doc/contributing.md)
- [x] `make` runs successfully
- [x] All tests in `./test.sh` pass
- [x] README and other documentation updated
- [x] [Pending changelog](./CHANGELOG_PENDING.md) updated

### CODE REVIEW

**Review by AlexKordic — APPROVED** [review]

File: doc/transcodingoptions.md [hunk 1]
```diff
@@ -89,21 +89,21 @@ An example of a full JSON configuration:
         "bitrate": 5000000,
         "fps": 30000,
         "fpsDen": 1001,
-        "profile": H264High,
-        "gop": "2",
+        "profile": "H264High",
```
Comment (AlexKordic) [inline doc/transcodingoptions.md:92]: Maybe [constant](https://github.com/livepeer/lpms/blob/master/ffmpeg/videoprofile.go#L40) should be lowercase? or it doesn't matter?

File: doc/transcodingoptions.md [hunk 2]
```diff
@@ -89,21 +89,21 @@ An example of a full JSON configuration:
...
-        "profile": H264ConstrainedHigh,
+        "profile": "H264ConstrainedHigh"
     },{
         "name":"highlight-reel",
         "width":160,
         "height":120,
         "bitrate":100000,
-        "fps": "0.1",
-        "gop":"intra",
+        "fps": 1,
```
Comment (cyberj0g) [inline doc/transcodingoptions.md:105]: Maybe original intention was to highlight that fractional FPS are supported? e.g. 24000/1001 and 30000/1001 are obsolete, but still quite common.

File: doc/transcodingoptions.md (hunk 1 above)
Comment (thomshutt) [reply doc/transcodingoptions.md:92]: We lowercase it, so I guess it doesn't matter https://github.com/livepeer/lpms/blob/f4dcebdca29cd2a0609d0317b1bd06e12af904bc/ffmpeg/videoprofile.go#L195

File: doc/transcodingoptions.md (hunk 2 above)
Comment (thomshutt) [reply doc/transcodingoptions.md:105]: Yeah, that could've been the original intent but looks like it's an int now https://github.com/livepeer/go-livepeer/blob/d655332b322e366cef7d1f629112c94e87a4d463/net/lp_rpc.pb.go#L1043

**Pull request merged**